src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -72,7 +72,7 @@ class MockWorkflowManagerActor extends Actor  {
       }
```

Comment:
I added a couple tests (one for call logs and one for workflow logs), but like you said it's mostly mocked out.
I think it's be nice to add one in StdoutStderrWorkflowSpec which actually run workflows.